src/nbl/video/CVulkanPhysicalDevice.h

```diff
@@ -284,7 +287,11 @@ class CVulkanPhysicalDevice final : public IPhysicalDevice
                 m_properties.limits.maxDescriptorSetUpdateAfterBindStorageImages            = descriptorIndexingProperties.maxDescriptorSetUpdateAfterBindStorageImages;
                 m_properties.limits.maxDescriptorSetUpdateAfterBindInputAttachments         = descriptorIndexingProperties.maxDescriptorSetUpdateAfterBindInputAttachments;
             }
-            
+            else
+            {
+                m_properties.limits.maxUpdateAfterBindDescriptorsInAllPools = ~0u;
```

Comment:
Let's only work with struct default values, because the code would be to long and we would need to do it for all backends.
I know I do the wrong thing and memset everything to 0 in IPhysicalDevice constructor, but that would get removed and basically every limit, feature member would have a default value.